lib/helpers.js

```diff
@@ -198,3 +199,14 @@ module.exports.getLetters = function(message) {
   return message.replace(/[^a-zA-Z]/g, '');
 }
 
+/**
+ * Returns hash for given string with DS API assword key. 
+ * @param {string} text
+ * @return {string}
+ */
+module.exports.generatePassword = function(text) {
```

Comment:
This function is copied from https://github.com/DoSomething/gambit/blob/e36bbd1928f12f865b42ebc00437f4bb6e9a13b1/lib/phoenix.js#L42 -- I didn't update it there because we're looking to deprecate that endpoint completely once Gambot goes live, and would be extra changes we'd need to test for this PR.